CONTRIBUTING.markdown

```diff
@@ -1,106 +1,124 @@
-Contribute
-==========
-
-So you've got an awesome idea to throw into Jekyll. Great! Please keep the
-following in mind:
-
-* **Use https://talk.jekyllrb.com for non-technical or indirect Jekyll questions that are not bugs.**
-* **Contributions will not be accepted without tests or necessary documentation updates.**
-* If you're creating a small fix or patch to an existing feature, just a simple
-  test will do. Please stay in the confines of the current test suite and use
-  [Shoulda](https://github.com/thoughtbot/shoulda/tree/master) and
-  [RSpec-Mocks](https://github.com/rspec/rspec-mocks).
-* If it's a brand new feature, make sure to create a new
-  [Cucumber](https://github.com/cucumber/cucumber/) feature and reuse steps
-  where appropriate. Also, whipping up some documentation in your fork's `site`
-  would be appreciated, and once merged it will be transferred over to the main
-  `site`, jekyllrb.com.
-* If your contribution changes any Jekyll behavior, make sure to update the
-  documentation. It lives in `site/_docs`. If the docs are missing information,
-  please feel free to add it in. Great docs make a great project!
-* Please follow the [GitHub Ruby Styleguide](https://github.com/styleguide/ruby)
-  when modifying Ruby code.
-* Please do your best to submit **small pull requests**. The easier the proposed
-  change is to review, the more likely it will be merged.
-* When submitting a pull request, please make judicious use of the pull request
-  body. A description of what changes were made, the motivations behind the
-  changes and [any tasks completed or left to complete](http://git.io/gfm-tasks)
-  will also speed up review time.
-
-Test Dependencies
------------------
-
-To run the test suite and build the gem you'll need to install Jekyll's
-dependencies. Simply run this command to get all setup:
+---
+layout: docs
+title: Contributing
+permalink: /docs/contributing/
+---
 
-    $ script/bootstrap
+# Contribute to Jekyll
 
-Before you start, run the tests and make sure that they pass (to confirm your
-environment is configured properly):
+Hi there! Interested in contributing to Jekyll? We'd love your help. Jekyll is an open source project, built one contribution at a time by users like you.
 
-    $ script/cibuild
+## Where to get help or report a problem
 
-If you are only updating a file in `test/`, you can use the command:
+* If you have a question about using Jekyll, start a discussion on https://talk.jekyllrb.com.
+* If you think you've found a bug within Jekyll itself, [open an issue](https://github.com/jekyll/jekyll/issues/new)
 
-    $ script/test test/blah_test.rb
+## Ways to contribute
 
-If you are only updating a `.feature` file, you can use the command:
+Whether you're a developer, a designer, or just a Jekyll devotee, there are lots of ways to contribute. Here's a few ideas:
```

Comment:
I'm a developer AND a Jekyll devotee! What does it mean to be a devotee? :smile: